### DISCUSSION TITLE
install without docker

### DISCUSSION THREAD

**Discussion by NathanPeake** [ZoeyVid/NPMplus#589] [Q&A] [answered]
install steps for without docker?

I use this for NPM [https://github.com/ej52/proxmox-scripts/tree/main/apps/nginx-proxy-manager](https://github.com/ej52/proxmox-scripts/tree/main/apps/nginx-proxy-manager)
whats the equivalent for NPMplus?

**Comment by Zoey2936** [comment, marked as answer]
just use docker
